indico/htdocs/sass/modules/_bootstrap.scss

```diff
@@ -0,0 +1,262 @@
+@import 'base';
+
+.bootstrap-header {
+    @extend %font-family-bootstrap;
+
+    font-weight: bolder;
+
+    z-index: 1;
+
+    display: block;
+
+    width: 100%;
+    height: 350px;
+    padding-top: 20px;
+
+    text-align: center;
+
```

Comment:
I find it very helpful as I don't have to go through the whole thing every time if I need to check, change or add something. And it's automatic :wink: